fk/keymaintain.go

```diff
@@ -345,6 +345,19 @@ func promptAndTurnOnMaintainAutomatically(prompter promptYesNoInterface, keyTask
 	out.Print("This requires storing the password in the system keyring.\n\n")
 
 	if prompter.promptYesNo(promptMaintainAutomatically, "", keyTask.key) == true {
+		if Keyring.ShouldInstructAboutPermissions() {
```

Comment:
Why only on macOS? I'd be perfectly happy to see it read back from my linux keyring, even if I didn't have to click "allow"

Reply:
I'm also not sure what other flavours of keyring are out there. Maybe on Arch it would prompt?

Reply:
I've modified this now and force pushed!